src/workerd/api/system-streams.c++

```diff
@@ -252,11 +258,87 @@ kj::Own<WritableStreamSink> newSystemStream(
   return kj::heap<EncodedAsyncOutputStream>(kj::mv(inner), encoding, context);
 }
 
+class RefcountedAsyncIoStream final : public kj::AsyncIoStream, public kj::Refcounted {
```

Comment:
This can be replaced with the new `kj::refcountedWrapper(...)` correct?

Reply:
That's the plan, see thread above: https://github.com/cloudflare/workerd/pull/237#discussion_r1066867895